app/presenters/users/edit_presenter.rb

```diff
@@ -8,16 +8,16 @@ def props
       {
         current_user_id: current_user.id,
         name: current_user.name,
+        preferred_name: current_user.preferred_name || '',
         about: current_user.about || '',
         locale: current_user.locale,
         available_locales: Rails.application.secrets.locale[:available],
         has_current_password: current_user.encrypted_password.present?,
-        avatar_url:
-          if current_user.avatar.attached?
-            current_user.avatar_url(variant: :mid)
-          else
-            nil
-          end,
+        avatar_url: if current_user.avatar.attached?
+          current_user.avatar_url(variant: :mid)
+        else
+          nil
+        end,
```

Comment:
Incorrect formatting, please use the [recommended config](https://docs.pupilfirst.com/developers/development_setup#code-formatting)